x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/middle_truncated_text.tsx

```diff
@@ -59,6 +80,10 @@ const StyledButton = styled(EuiButtonEmpty)`
   }
 `;
 
+const SecureIcon = euiStyled(EuiIcon)`
+  margin-right: 3px;
```

Comment:
Similar as suggested above to use the provided EUI size variables instead. Changes it to `4px` but visually not much difference. Not sure if this implicates some width calcs somewhere?

```suggestion
  margin-right: ${(props) => props.theme.eui.euiSizeXS};
```